desktop/dev-environments.md

```diff
@@ -243,9 +243,19 @@ The `development`target uses a `golang:1.16-alpine` image with all dependencies
 
 In our example, the Docker Compose files are the same. However, they could be different and the services defined in the main Compose file may use other targets to build or directly reference other images.
 
+## Specify a Dockerfile 
+
+In this preview, Dev Environments support a simple YAML file which allows you to specify a Dockerfile to define your Dev Environment. You must include this as part of the `.docker` folder and then add it as a `config.json` file. For example:
+
+```jsx
```

Comment:
s/jsx/json/ ? (not sure if rouge  understands jsx)

Reply:
Looks like we already used `jsx` in the [base image example](https://github.com/docker/docker.github.io/pull/13973/files#diff-1c418e0ee39feb65f4c22b9a4dca3b1d4723f00e3140a26cf909c0effebc3815R260)